debug.log

```diff
@@ -0,0 +1,2 @@
+[1117/092646.211:ERROR:directory_reader_win.cc(43)] FindFirstFile: Maaritettya polkua ei loydy. (0x3)
```

Comment:
please add debug.log to .gitignore. Additionally, if possible remove file from git (as it is already part of git repo).